examples/c/xmake.lua

```diff
@@ -0,0 +1,40 @@
+add_rules("mode.release", "mode.debug")
+add_rules("platform.linux.bpf")
+set_license("GPL-2.0")
+
+add_requires("linux-tools", {configs = {bpftool = true}})
+add_requires("libbpf")
```

Comment:
Does this link to the system libbpf? @anakryiko asked that the examples statically link to the vendored libbpf in this repo so users don't need to download any packages to get a simple example going.

Reply:
add_requires will first automatically detect the packages installed on the system, if they are not found, xmake will  download them automatically, and if they are found, they will be used directly.

Reply:
I saw your build output, it should have automatically detected libbpf on your system, so xmake did not install it.

Reply:
Of course, I can also switch to submodudle/libbpf and local bpftool without using add_requires, if you want.

But for building android programs, I suggest adding remote download, because it may be troublesome for users to install the environment by themselves.

Reply:
> Of course, I can also switch to submodudle/libbpf and local bpftool without using add_requires, if you want.

Yeah, I think that would be preferable. At least for libbpf-bootrap. But for "real" applications, xmake should also provide the ability to link to system libs, as distro packagers don't like vendored libraries.

Reply:
Ok, I will switch to local libpbf sources.